.github/workflows/ci.yml

```
@@ -42,3 +42,7 @@ jobs:
         DB_USERNAME: postgres
         REDIS_HOST: redis
         REDIS_PORT: ${{ job.services.redis.ports[6379] }}
+    - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1.0.7
+        with:
+          file: ./coverage/coverage.xml
```

```suggestion
      uses: codecov/codecov-action@v1.0.7
      with:
        file: ./coverage/coverage.xml
```

This was erroring: https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/actions/runs/123850026

---

😱 have updated!